corehq/apps/userreports/reports/specs.py

```diff
@@ -186,8 +186,27 @@ def get_column_config(self, data_source_config, lang):
     def get_fields(self, data_source_config=None, lang=None):
         return [self.field]
 
+    def _data_source_col_config(self, data_source_config):
+        return filter(
+            lambda c: c['column_id'] == self.field, data_source_config.configured_indicators
+        )[0]
```

Comment:
Yeah that should be fine because if this fails it's an invalid report configuration. It does seem like brittle code though